build/android/gyp/apkbuilder.py

```diff
@@ -234,8 +234,8 @@ def _AddFiles(apk, details):
     try:
       apk.getinfo(apk_path)
       # Should never happen since write_build_config.py handles merging.
-      raise Exception(
-          'Multiple targets specified the asset path: %s' % apk_path)
+      # raise Exception(
```

Comment:
@kaidokert there's an issue here with the starboard toolchain using the same .so as the default toolchain that I'm not sure how to resolve—I could dig into write_build_config.py further to see if there's some configuration I'm missing, or leave it as is for now